public/js/index.js

```diff
@@ -75,25 +35,55 @@ rates = {
   }
   
   window.addEventListener('load', function() {
-    
+
+    const d = new Date();
+    let text = d.toLocaleString();
+    document.getElementById("demo").innerHTML = text;
+
     const form = document.getElementById("my-form"); 
     form.addEventListener("submit", handleSubmit);
-    let arr = [
-      { symbol: '$', currency: 'Dollar', buy: '37.00', sell: '37.45' },
-      { symbol: '&#8364', currency: 'Euro', buy: '40.00', sell: '40.65' },
-      { symbol: '&#163', currency: 'Pound', buy: '46.15', sell: '46.95' },
-      { symbol: '&#65509', currency: 'Yuan', buy: '5.05', sell: '5.45' },
-      { symbol: 'z&#x142', currency: 'Zloty', buy: '9.20', sell: '9.35' },
-    ];
-  
-    arr.forEach(function(item) {
-      let row = document.createElement('tr')
-      row.innerHTML = `
-      <td>${item.symbol}</td>
-      <td>${item.currency}</td>
-      <td>${item.buy}</td>
-      <td>${item.sell}</td>
-      `
-      document.querySelector('table').append(row)
-    });
-  });
\ No newline at end of file
+
+    let interval = setInterval(() => {
+    const url = 'http://localhost:3000/api/v1/currencies';
+    const xhr = new XMLHttpRequest();
+    xhr.open('GET', url,);
+    xhr.onload = () => {
+      console.log(xhr.response);
+      arr = JSON.parse(xhr.responseText);
+
+      arr.forEach(function(item) {
+        let row = document.createElement('tr');
+        row.innerHTML = `
+        <td>${item.symbol}</td>
+        <td>${item.currency}</td>
+        <td>${item.buy}</td>
+        <td>${item.sell}</td>
+        `
+        document.querySelector('table').append(row);
+      });
+    }
+    xhr.send();
+  }, 60000)
+
+  const url = 'http://localhost:3000/api/v1/currencies';
```

Comment:
this code is the same ;) 
write a function and call it from both places

Reply:
Ready!